crates/project/src/project_settings.rs

```diff
@@ -101,6 +101,9 @@ pub struct LspSettings {
     pub binary: Option<BinarySettings>,
```

Comment:
I think we should make the new setting part of `binary` and name it `pathLookup` (it being `pathLookup: Option<bool>`). By default it's `None` and if it's `None`, then the language extension decides what the best default is (for `gopls`, for example, it does the lookup, for Rust it doesn't).

Users can then opt-in/out by setting it to `true`/`false`.

And if a `binary.path` is set, that overwrites the behaviour of `binary.pathLookup`.

Reply:
Okay, did just that